src/Common/src/Interop/Windows/Winsock/Interop.GetAddrInfoExW.cs

```diff
@@ -29,8 +29,8 @@ internal static partial class Winsock
             [Out] out IntPtr lpNameHandle
         );
 
-        [DllImport("ws2_32.dll", ExactSpelling = true, SetLastError = true)]
-        internal static extern unsafe void FreeAddrInfoEx([In] AddressInfoEx* pAddrInfo);
+        [DllImport("ws2_32.dll", ExactSpelling = true)]
```

Comment:
One more nit: This can use `Interop.Libraries.Ws2_32`

Reply:
Ugh. Yup.